misc/angular-app.misc.md

```diff
@@ -1,56 +1,62 @@
-## angular-app thoughts
+## Review of angular-app
```

Comment:
Do we still need this now that we have the issues?

Reply:
I guess not. I copied relevant parts to issues.